sonar-ws/src/main/gen-java/org/sonarqube/ws/Rules.java

```diff
@@ -302,24 +272,9 @@ private Rule(
                 org.sonarqube.ws.Rules.ListResponse.Rule.class, org.sonarqube.ws.Rules.ListResponse.Rule.Builder.class);
       }
 
-      public static com.google.protobuf.Parser<Rule> PARSER =
-          new com.google.protobuf.AbstractParser<Rule>() {
-        public Rule parsePartialFrom(
-            com.google.protobuf.CodedInputStream input,
-            com.google.protobuf.ExtensionRegistryLite extensionRegistry)
-            throws com.google.protobuf.InvalidProtocolBufferException {
-          return new Rule(input, extensionRegistry);
-        }
-      };
-
-      @java.lang.Override
-      public com.google.protobuf.Parser<Rule> getParserForType() {
-        return PARSER;
-      }
-
       private int bitField0_;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this field "bitField0_" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS00116)